generator/mavgen_python.py

```diff
@@ -11,8 +11,11 @@
 
 import os
 import sys
+import operator
 import textwrap
 from . import mavtemplate
+from . import mavparse
```

Comment:
Don't know how these changes got here. I don't think needed, but need to retest.